common-lib/src/main/scala/com/gu/mediaservice/lib/imaging/ImageOperations.scala

```diff
@@ -114,31 +116,37 @@ class ImageOperations(playPath: String) extends GridLogging {
   val thumbUnsharpRadius = 0.5d
   val thumbUnsharpSigma = 0.5d
   val thumbUnsharpAmount = 0.8d
-  def createThumbnail(sourceFile: File, sourceMimeType: Option[MimeType], width: Int, qual: Double = 100d,
-                      tempDir: File, iccColourSpace: Option[String], colourModel: Option[String]): Future[File] = {
+  def createThumbnail(sourceFile: File,
```

Comment:
As you've been through this enough to understnad the flow to some degree would it be possible to add some basic scaladoc to these methods?